core/src/main/java/org/infinispan/reactive/publisher/impl/commands/batch/PublisherResponseExternalizer.java

```diff
@@ -0,0 +1,90 @@
+package org.infinispan.reactive.publisher.impl.commands.batch;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Set;
+
+import org.infinispan.commons.io.UnsignedNumeric;
+import org.infinispan.commons.marshall.AbstractExternalizer;
+import org.infinispan.commons.marshall.Ids;
+import org.infinispan.commons.util.IntSet;
+import org.infinispan.commons.util.Util;
+
+public class PublisherResponseExternalizer extends AbstractExternalizer<PublisherResponse> {
```

Comment:
Should this be an internal class of `PublisherResponse` like we do elsewhere?

Reply:
It is used by both `PublisherResponse` and `KeyPublisherResponse` which is why I separate it.